packages/node_modules/@webex/internal-plugin-calendar/test/integration/spec/calendar.js

```diff
@@ -279,5 +323,112 @@ describe.skip('plugin-calendar', () => {
           });
       });
     });
+    describe('#meetingNotes', () => {
+      let creator, mccoy, webex, spock;
+
+      before('create test users', () => testUsers.create({
+        count: 3,
+        config: {
+          entitlements: [
+            'webex',
+            'squaredCallInitiation',
+            'squaredInviter',
+            'squaredRoomModeration',
+            'webExSquared',
+            'squaredFusionCal'
+          ]
+        }
+      })
+        .then((users) => {
+          console.log('Enter then');
+          [creator, spock, mccoy] = users;
+          console.log('Got creator, spock, and mccoy');
+
+          webex = new WebexCore({
+            credentials: {
+              authorization: creator.token
+            },
+            config: {
+              services: {
+                discovery: {
+                  whistlerServiceUrl: process.env.WHISTLER_API_SERVICE_URL || 'https://calendar-whistler.onint.ciscospark.com:8084/api/v1'
```

Comment:
Is this service not provided by the U2C catalog? Our tests could run on multiple environments.

Reply:
I'm not very sure of how the whistler service is working, so I just used the existing code as reference when writing my own test cases. I'll check if the catalog can be used instead.

Reply:
It doesn't look like whistler is a service provided by U2C, so we should be ok with hard coding endpoints, but wary that they could be the cause of flakiness.